app/controllers/registration/business/EuGoodsController.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.registration.business
+
+import config.{AuthClientConnector, FrontendAppConfig}
+import connectors.KeystoreConnector
+import controllers.BaseController
+import forms.EuGoodsForm
+import javax.inject.{Inject, Singleton}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.{ApplicantDetailsService, SessionProfile, TradingDetailsService}
+import views.html.{eu_goods => EuGoodsPage}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class EuGoodsController @Inject()(mcc: MessagesControllerComponents,
+                                  val keystoreConnector: KeystoreConnector,
+                                  val authConnector: AuthClientConnector,
+                                  val applicantDetailsService: ApplicantDetailsService,
+                                  val tradingDetailsService: TradingDetailsService)
+                                 (implicit val appConfig: FrontendAppConfig,
+                                  val executionContext: ExecutionContext) extends BaseController(mcc) with SessionProfile {
+
+  val show: Action[AnyContent] = isAuthenticatedWithProfile {
+    implicit request =>
+      implicit profile =>
+        tradingDetailsService.getTradingDetailsViewModel(profile.registrationId) map {
+          _.euGoods match {
+            case Some(goods) => Ok(EuGoodsPage(EuGoodsForm.form.fill(goods)))
```

Comment:
Is there a reason why this view isn't injected but the others are?

Reply:
My PR isn't touching that page, the EU Goods controller is merely what used to be called the "TradingDetailsController" less the trading name routes

Reply:
i.e. out of scope for this (already scope-stretched) ticket